test/jdk/java/lang/reflect/code/type/TestJavaType.java

```diff
@@ -177,11 +177,11 @@ public void testTypeRoundTrip(Type type) throws ReflectiveOperationException {
         Assert.assertEquals(javaType, CoreTypeFactory.JAVA_TYPE_FACTORY.constructType(javaType.externalize()));
     }
 
-    @Test(dataProvider = "types")
-    public void testTypeString(Type type) throws ReflectiveOperationException {
-        JavaType javaType = JavaType.type(type);
-        Assert.assertEquals(type.getTypeName(), javaType.toString());
-    }
+//    @Test(dataProvider = "types")
```

Comment:
I've decided to comment this test -- since it relies on the assumption that `toString` of a `JavaType` is exactly the same as `getTypeName` of `j.l.r.Type`. While this is generally true, it doesn't quite hold for type-variables.